### PR TITLE
[SWPWA-362] Promotion note for tier price

### PR DESCRIPTION
**Pull request by yeegor**
Implemented tier pricing related frontend functionality

### CODE REVIEW

File: src/app/component/ProductActions/ProductActions.style.scss [hunk 1]
```diff
@@ -322,4 +322,16 @@
...
+
+    &-TierPrices {
+        @include after-mobile {
+            order: 24;
+            margin-top: 3%;
+        }
+        @include mobile {
+            order: -1;
+            margin-left: 10px;
+            margin: auto;
```
Comment (Danpiel) [inline src/app/component/ProductActions/ProductActions.style.scss]: Codacy Issue found: [Unexpected shorthand "margin" after "margin-left" (declaration-block-no-shorthand-property-overrides)](https://app.codacy.com/manual/ScandiPWA/base-theme/pullRequest?prid=4869008)

**Comment by Danpiel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           

Complexity increasing per file
==============================
- src/app/component/TierPrices/TierPrices.component.js  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/ScandiPWA/base-theme/pullRequest?prid=4869008&bid=16144555)

File: src/app/component/TierPrices/TierPrices.style.scss [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+        &:last-child {
+            padding-bottom: 0;
+        }
+
+        &::before {
+            content: none;
+        }
+    }
+}
```
Comment (Danpiel) [inline src/app/component/TierPrices/TierPrices.style.scss]: Codacy Issue found: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/manual/ScandiPWA/base-theme/pullRequest?prid=4869008)

**Comment by Danpiel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- src/app/component/TierPrices/TierPrices.component.js  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/ScandiPWA/base-theme/pullRequest?prid=4869008&bid=16144555)

File: src/app/component/TierPrices/TierPrices.style.scss (hunk 2 above)
Comment (Danpiel) [inline src/app/component/TierPrices/TierPrices.style.scss]: Codacy Issue found: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/manual/ScandiPWA/base-theme/pullRequest?prid=4869008)

File: src/app/component/ProductActions/ProductActions.component.js [hunk 3]
```diff
@@ -32,6 +32,7 @@ import Html from 'Component/Html';
 import Link from 'Component/Link';
 
 import './ProductActions.style';
+import TierPrices from 'Component/TierPrices';
```
Comment (alfredsgenkins) [inline src/app/component/ProductActions/ProductActions.component.js]: Please order imports:
- node modules
- aliases
- local (path)

File: src/app/component/ProductActions/ProductActions.component.js [hunk 4]
```diff
@@ -331,6 +333,15 @@ export default class ProductActions extends PureComponent {
         );
     }
 
+    renderTierPrices() {
+        const { productOrVariant } = this.props;
+        return (
```
Comment (alfredsgenkins) [inline src/app/component/ProductActions/ProductActions.component.js]: Prefer enter between `return` and `const` declaration

File: src/app/component/TierPrices/TierPrices.style.scss (hunk 2 above)
Comment (alfredsgenkins) [inline src/app/component/TierPrices/TierPrices.style.scss]: Added new line in the end of the file ;)

File: src/app/type/ProductList.js [hunk 5]
```diff
@@ -118,7 +118,11 @@ export const ProductType = PropTypes.shape({
     special_price: PropTypes.number,
     thumbnail: PropTypes.shape({ url: PropTypes.string }),
     thumbnail_label: PropTypes.shape({ label: PropTypes.string }),
-    tier_prices: PropTypes.string,
+    tier_prices: PropTypes.arrayOf(PropTypes.shape({
+        value: PropTypes.number.isRequired,
```
Comment (alfredsgenkins) [inline src/app/type/ProductList.js]: Do not put `isRequired` here, this is overall type, people might request those, might not :)

**Comment by alfredsgenkins** [comment]
Please add https://github.com/scandipwa/catalog-graphql/releases/tag/2.6.5 dependency.

**Comment by negzu** [comment]
Corrections have been applied.

**Comment by Danpiel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- src/app/component/TierPrices/TierPrices.component.js  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/ScandiPWA/base-theme/pullRequest?prid=4869008&bid=16144555)

File: src/app/component/TierPrices/TierPrices.style.scss (hunk 2 above)
Comment (Danpiel) [inline src/app/component/TierPrices/TierPrices.style.scss]: Codacy Issue found: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/manual/ScandiPWA/base-theme/pullRequest?prid=4869008)

**Comment by alfredsgenkins** [comment]
Please make sure ti depend on `2.6.6`. Make sure to use Magento 2 default.

**Pull request merged**